tests/test_auto_offer_on_device_rest.py

```diff
@@ -3,10 +3,24 @@
 """
 import logging
 import sys
+import argparse
+import numpy as np
 from time import sleep
+from random import randint
 from d3a_api_client.rest_device import RestDeviceClient
 from d3a_api_client.utils import get_area_uuid_from_area_name_and_collaboration_id
 
+#parse arguments
+parser = argparse.ArgumentParser()
+parser.add_argument("--simulation_id", type=str, help="Simulation uuid")
+parser.add_argument("--load", type=str, help="Load device")
```

Comment:
Please be more descriptive here. It is really a Load ID,  right?

Reply:
Actually the argument that needs to be passed is 'Load' or 'PV'. Maybe a boolean flag would be better ? Or considering here that the devices are always Load and PV, should it be "hard-coded" ?